models/training-models/diabetes-prediction/02-model-training-tensorflow.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Model Training with TensorFlow\n",
+    "\n",
+    "Train a model that predicts whether or not a patient has diabetes, based on medical features. \n",
+    "\n",
+    "### 1. Import the required libraries and packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from typing import List, Dict\n",
+    "\n",
+    "import pandas as pd\n",
+    "\n",
+    "import tensorflow as tf\n",
+    "\n",
+    "import numpy as np\n",
+    "\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import classification_report"
+   ]
```

Comment:
We should warn the student about the resulting warning.

Reply:
Added a brief explanation about these warnings. Students can ignore them.